dev-libs/wlc/wlc-0.0.2-r1.ebuild

```diff
@@ -48,3 +48,11 @@ src_configure() {
 
 	cmake-utils_src_configure
 }
+
+pkg_postinst() {
+	if use X
```

Comment:
You may also want to use `has_version 'x11-base/xorg-server[wayland]'` to check if xorg-server is already built correctly, and display the message only if it isn't.